api/db-cdr/changelog/db.changelog-2.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="brianfreeman" id="changelog-2" runAlways="true">
+
+    <createTable tableName="criteria">
+      <column name="id" type="BIGINT" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="sort_order" type="BIGINT">
+        <constraints nullable="false"/>
+      </column>
+      <column name="parent_id" type="BIGINT">
+        <constraints nullable="false"/>
+      </column>
+      <column name="type" type="VARCHAR(20)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="code" type="VARCHAR(80)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="is_group" type="BOOL">
+        <constraints nullable="true"/>
+      </column>
+      <column name="is_selectable" type="BOOL">
+        <constraints nullable="true"/>
+      </column>
+      <column name="est_count" type="VARCHAR(80)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="domain_id" type="VARCHAR(80)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="concept_id" type="VARCHAR(80)">
```

Comment:
Random thought... should we have a concept table that this would be a foreign key to? (I guess we don't need to for now?)

Reply:
not sure we need it in Cloud Sql yet, but this concept_id will help us map to concept table in BQ